node/flatpak-node-generator.py

```diff
@@ -1377,7 +1377,8 @@ def _finalize(self) -> None:
                     original_version = f'{source.original}'
                     new_version = f'{path}#{source.commit}'
                     assert source.from_ is not None
-                    data['package.json'][source.from_] = new_version
+                    from_ = re.sub(r'^[^@]+@git\+', 'git+', source.from_)
```

Comment:
minor nit: it would be nice if this were `re.compile`'d in a constant, and I *think* you can avoid repeating `git+` in the replace arg by making the regex `^[^@]+(?=git\+)`